code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -205,12 +226,49 @@ Difficulty: Very Hard
 		var/turf/E = get_step(src, d)
 		shoot_projectile(E)
 
-/mob/living/simple_animal/hostile/megafauna/colossus/proc/telegraph()
+/mob/living/simple_animal/hostile/megafauna/colossus/proc/telegraph(mode)
 	for(var/mob/M in range(10, src))
 		M.flash_screen_color("#C80000", 1)
 		shake_camera(M, 4, 3)
 	playsound(src, 'sound/magic/narsie_attack.ogg', 200, TRUE)
+	if(mode)
+		say("<span class='colossus'>\"<b>[mode]</b>\"</span>", sanitize = FALSE, ignore_speech_problems = TRUE, ignore_atmospherics = TRUE)
```

Comment:
Honestly, why not just override colossus' `say()` to put these arguments in by default? Seems like a lot of extra cruft

Reply:
done